src/Pipfile

```diff
@@ -5,7 +5,12 @@ name = "pypi"
 
 [packages]
 antlr4-python3-runtime = "==4.7.2"
+bcrypt = "==4.1.2"
```

Comment:
So my understanding is that only direct dependencies are supposed to go in this file. But I think youre adding indirect ones right? Could you find out what is the correct practice?

Reply:
I am adding the indirect ones which are C extensions, and that is intentional. I want to be absolutely sure the binary wheels will be available once we do the version bump, and if I do not pin these, when we lock the dependencies it is possible that pip will resolve some version that makes it harder to upgrade (or introduces regressions).

Reply:
OK